DEPS

```diff
@@ -28,7 +28,7 @@ vars = {
   # This prevents us from downloading the Emscripten toolchain for builds
   # which do not build for the web. This toolchain is needed to build CanvasKit
   # for the web engine.
-  'download_emsdk': False,
+  'download_emsdk': True,
```

Comment:
It could be useful to disable for some builders

Reply:
Definitely useful to avoid extra work for builders that don't need it.

I'm curious if this is the right time to default it to true. People working on this can add the varaible to their .gclient file for now to make sure they get it. If we flip this it will increase CI time across the board for engine builds without some recipe changes (unless that's already landed somewhere?).